protobuf-maven-plugin/src/main/java/io/github/ascopes/protobufmavenplugin/sources/ProtoArchiveExtractor.java

```diff
@@ -113,8 +113,15 @@ private FileSystem openZip(Path path) throws IOException {
     // Impl note: unlike other constructors, calling this multiple times on the same path
     // will open multiple file system objects. Other constructors do not appear to do this,
     // so would not be thread-safe for concurrent plugin executions.
-    return FileUtils.getFileSystemProvider("jar")
-        .newFileSystem(path, Map.of());
+    try {
+      return FileUtils.getFileSystemProvider("jar")
+          .newFileSystem(path, Map.of());
+
+    } catch (RuntimeException ex) {
```

Comment:
```suggestion
    } catch (Exception ex) {
```
Probably worth rethrowing IO exceptions as well.